.viperlightrc

```diff
@@ -0,0 +1,4 @@
+{
+  "all": true,
```

Comment:
Are you aware the Viperlight config will only be used in case of a local scan?
Our builder pipeline ignores .viperlightrc and always uses the default settings.